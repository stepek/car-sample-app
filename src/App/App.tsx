import React from "react"
import {Provider} from "react-redux"

import CarInfoPage from "../modules/carInfo/components/CarInfoPage/CarInfoPage"
import {createStore} from "../modules/createStore"

export function App() {
  const store = createStore()

  return (
    <Provider store={store}>
      <CarInfoPage />
    </Provider>
  )
}

export default App
