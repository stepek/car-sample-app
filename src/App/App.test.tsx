import React from "react"
import renderer from "react-test-renderer"

import App from "./App"

test("<App>", () => {
  const component = renderer.create(<App />)

  expect(component.toJSON()).toMatchSnapshot()
})
