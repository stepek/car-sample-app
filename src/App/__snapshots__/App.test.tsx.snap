// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> 1`] = `
<div
  className="rc-select select-0-2-1 rc-select-single rc-select-show-arrow"
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
>
  <div
    className="rc-select-selector"
    onClick={[Function]}
    onMouseDown={[Function]}
  >
    <span
      className="rc-select-selection-search"
    >
      <input
        aria-activedescendant="undefined_list_0"
        aria-autocomplete="list"
        aria-controls="undefined_list"
        aria-haspopup="listbox"
        aria-owns="undefined_list"
        autoComplete="off"
        className="rc-select-selection-search-input"
        onChange={[Function]}
        onCompositionEnd={[Function]}
        onCompositionStart={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onPaste={[Function]}
        readOnly={true}
        role="combobox"
        style={
          Object {
            "opacity": 0,
          }
        }
        type="search"
        unselectable="on"
        value=""
      />
    </span>
    <span
      className="rc-select-selection-placeholder"
    >
      Select Car Maker
    </span>
  </div>
  <span
    aria-hidden={true}
    className="rc-select-arrow"
    onMouseDown={[Function]}
    style={
      Object {
        "WebkitUserSelect": "none",
        "userSelect": "none",
      }
    }
    unselectable="on"
  >
    <span
      className="rc-select-arrow-icon"
    />
  </span>
</div>
`;
